front/app/containers/Admin/communityMonitor/routes.tsx

```diff
@@ -1,42 +1,97 @@
 import React, { lazy } from 'react';
 
-import { Outlet as RouterOutlet } from 'react-router-dom';
+import { Navigate } from 'react-router-dom';
 
-import HelmetIntl from 'components/HelmetIntl';
 import PageLoading from 'components/UI/PageLoading';
 
+const LiveMonitor = lazy(() => import('./components/LiveMonitor'));
+const Reports = lazy(() => import('./components/Reports'));
+const Settings = lazy(() => import('./components/Settings/Index'));
+const Participants = lazy(() => import('./components/Participants'));
+
 import { AdminRoute } from '../routes';
 
 import CommunityMonitorSurveyFormBuilder from './CommunityMonitorFormBuilder';
-import messages from './messages';
 
 const CommunityMonitor = lazy(() => import('./index'));
 
 export enum communityMonitorRoutes {
   communityMonitor = 'community-monitor',
   communityMonitorSurveyEdit = 'projects/:projectId/phases/:phaseId/survey/edit',
   communityMonitorDefault = '',
+  liveMonitor = 'live-monitor',
+  participants = 'participants',
+  participantsProjects = 'participants/projects/:projectId',
+  reports = 'reports',
+  settings = 'settings',
 }
 
 export type communityMonitorRouteTypes =
   | AdminRoute<communityMonitorRoutes.communityMonitor>
+  | `${AdminRoute<communityMonitorRoutes.communityMonitor>}/${communityMonitorRoutes.liveMonitor}`
+  | `${AdminRoute<communityMonitorRoutes.communityMonitor>}/${communityMonitorRoutes.participants}/projects/${string}`
+  | `${AdminRoute<communityMonitorRoutes.communityMonitor>}/${communityMonitorRoutes.reports}`
+  | `${AdminRoute<communityMonitorRoutes.communityMonitor>}/${communityMonitorRoutes.settings}`
   | `${AdminRoute<communityMonitorRoutes.communityMonitor>}/projects/${string}/phases/${string}/survey/edit`;
 
 const communityMonitorsRoutes = () => {
   return {
     path: communityMonitorRoutes.communityMonitor,
     element: (
       <PageLoading>
-        <HelmetIntl title={messages.communityMonitorLabel} />
-        <RouterOutlet />
+        <CommunityMonitor />
       </PageLoading>
     ),
     children: [
       {
-        path: communityMonitorRoutes.communityMonitorDefault,
+        path: '',
+        element: <Navigate to="live-monitor" replace />,
+      },
+      {
+        path: communityMonitorRoutes.liveMonitor,
+        element: (
+          <PageLoading>
+            <div
+              role="tabpanel"
+              aria-labelledby="tab-live-monitor"
+              tabIndex={0}
+            >
```

Comment:
Would be good if we can make a reusable "TabPanel" + add this at the component level. We don't normally add code to routes, except for PageLoading and the parent component.